lib/usdUfe/ufe/UsdUndoableSetSceneItemMetadataCommand.cpp

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright 2023 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "UsdUndoableSetSceneItemMetadataCommand.h"
+
+#include "Utils.h"
+#include "private/Utils.h"
+
+#include <pxr/base/tf/diagnostic.h>
+#include <pxr/base/tf/token.h>
+#include <pxr/base/vt/dictionary.h>
+#include <pxr/base/vt/value.h>
+
+#include <ufe/scene.h>
+#include <ufe/sceneNotification.h>
+#include <ufe/undoableCommand.h>
+
+#include <utility>
```

Comment:
Is this needed?

Reply:
I think this is the remnant of another project that has clang tidy to disallow passing const references to constructors and instead to pass by value and do a std::move (from <utility>). 

Since we don't have that restriction, I'll remove the <utility>.